### PR TITLE
Fix some more minor misspellings to smooth it out

### PR DESCRIPTION
**Pull request by jeffadev**
Hey, back again.

This time I went through the entire locale YML file to spot any off-puttings. This is what I came up with so far. Hopefully for the English speakers the messages will be much smoother.

Thanks!

### CODE REVIEW

File: res/yaml/locale_EN.yml [hunk 1]
```diff
@@ -23,36 +23,36 @@ Headers:
...
+  databaseImportSuccess: "<yellow>The <gold><value><yellow> data have been successfully imported into your current database."
+  deadHorsesCleared: "<yellow>All your dead horses have been cleared from the database."
+  deadHorsesClearedOther: "<yellow>All <gold><player><yellow>'s dead horses have been cleared from the database."
   favoriteAlreadySet: "<yellow>You have already defined <gold><horse><yellow> as favorite horse."
   favoriteAlreadySetOther: "<gold><player><yellow> has already defined <gold><horse><yellow> as favorite horse."
   favoriteEdited: "<gold><horse><yellow> is now your favorite horse."
   favoriteEditedOther: "<gold><horse><yellow> is now the favorite horse of <gold><player><yellow>."
   friendAdded: "<gold><player><yellow> has been added to your friends list."
-  friendAddedReverse: "<gold><player><yellow> has added you in his friends list."
+  friendAddedReverse: "<gold><player><yellow> has added you to their friends list."
```
Comment (Zedd7) [inline res/yaml/locale_EN.yml:47]: Are you sure about this ? I always wondered if "friends list" was correct but here `<player>` refers to a single player. Thus, 'their' seems to be inappropriate.

File: res/yaml/locale_EN.yml [hunk 2]
```diff
@@ -23,36 +23,36 @@ Headers:
...
-  friendListReverseOther: "<yellow>Friends lists to which <gold><player><yellow> belongs : <value>"
+  friendList: "<yellow>Your friends list: <value>"
+  friendListOther: "<gold><player><yellow>'s friend list: <value>"
+  friendListReverse: "<yellow>Friends lists to which you belong: <value>"
+  friendListReverseOther: "<yellow>Friends lists to which <gold><player><yellow> belongs: <value>"
   friendRemoved: "<gold><player><yellow> has been removed from your friends list."
   friendRemovedReverse: "<gold><player><yellow> has removed you from his friends list."
   horseAlreadyClaimed: "<yellow>You have already claimed this horse."
-  horseAlreadyOnSale: "<gold><horse><yellow> is already on sale."
+  horseAlreadyOnSale: "<gold><horse><yellow> is already for sale."
```
Comment (Zedd7) [inline res/yaml/locale_EN.yml]: `horseAlreadyOnSale` should then be changed to `horseAlreadyForSale`

File: res/yaml/locale_EN.yml [hunk 3]
```diff
@@ -166,7 +166,7 @@ Economy:
   notEnoughMoney: "<yellow>You don't have <gold><currency><amount><yellow>."
 
 Horse informations:
-  health: "<yellow>Health : <gold><amount><yellow>/<gold><max>"
+  health: "<yellow>Health: <gold><amount><yellow>/<gold><max>"
```
Comment (Zedd7) [inline res/yaml/locale_EN.yml]: I would rather remove the space for all `Horse informations` indexes

**Review by Zedd7 — CHANGES_REQUESTED** [review]
Thanks again ! Please take a look at this review because I'm not sure about a few things

File: res/yaml/locale_EN.yml (hunk 1 above)
Comment (jeffadev) [reply res/yaml/locale_EN.yml:47]: Hey, this looks correct in English. We consider "his" to mean a male. By using "their", this means they could either be of Male or Female gender.

Stating the following, for example: "vueu has added you to their friends list.", looks completely proper!

File: res/yaml/locale_EN.yml (hunk 2 above)
Comment (jeffadev) [reply res/yaml/locale_EN.yml]: Yes, I did notice this, however I saw no breaking change in the plugin's functionality changing just the chat message. But if you'd like to keep consistency I will gladly go through and change the references in the code too.

**Comment by jeffadev** [comment]
There, please make sure I did not destroy anything with that commit. :) 👍

File: res/yaml/locale_EN.yml (hunk 2 above)
Comment (Zedd7) [reply res/yaml/locale_EN.yml]: Oh no don't worry about that, this message was for me^^
EDIT : Well, you already figured out ;)

**Review by Zedd7 — APPROVED** [review]

**Comment by Zedd7** [comment]
Thank you ! It feels good to know that the EN locale is now correct :)

**Comment by jeffadev** [comment]
Very glad I could help.

**Pull request merged**